tests/test_qvm.py

```diff
@@ -964,17 +969,17 @@ def test_compiled_program_was_correct_compared_with_default_qubit(self, qvm, dev
 
         results2 = qnodes2(params)
 
-        assert np.allclose(results, results2, atol=1e-02, rtol=0)
+        assert np.allclose(results, results2, atol=2e-02, rtol=0)
         assert dev.circuit_hash in dev._compiled_program_dict
         assert len(dev._compiled_program_dict.items()) == 1
 
-    @flaky(max_runs=10, min_passes=1)
+    @flaky(max_runs=5, min_passes=3)
```

Comment:
While not best practice for unittests, we've found flaky really nice for stochastic quantum integration tests :laughing:

Reply:
Seems like a nice tool!